### ISSUE TITLE
Temporary directories are not removed

### ISSUE THREAD

**Issue by ubruhin** [dbrgn/pydigitize#8]
Not sure if this is a real issue, but it seems that the created temporary directories are not removed after the script exits, which I don't like ;)

**Comment by dbrgn** [comment]
Yeah, so far I didn't delete them for debugging and because `/tmp` is usually deleted on reboot anyways. Don't you have your `/tmp` on a tmpfs?

**Comment by ubruhin** [comment]
> Don't you have your /tmp on a tmpfs?

I'm not sure, `mount` does not list `/tmp`, so probably not(?). But IMHO it would be anyway better to explicitly remove temporary directories, regardless of whether the system does that on reboot or not...

**Comment by dbrgn** [comment]
Yep, you're right :)
